test/e2e/common/node/sysctl.go

```diff
@@ -113,21 +114,21 @@ var _ = SIGDescribe("Sysctls [LinuxOnly] [NodeFeature:Sysctls]", func() {
 
 	/*
 	  Release: v1.21
-	  Testname: Sysctl, allow specified unsafe sysctls
-	  Description: Pod is created with kernel.shm_rmid_forced. Should allow unsafe sysctls that are specified.
+	  Testname: Sysctl, forbidden unsafe sysctls
+	  Description: Pod is created with net.core.somaxconn. Should forbidden unsafe sysctls as it is not configured in kubelet by default.
 	  [LinuxOnly]: This test is marked as LinuxOnly since Windows does not support sysctls
 	*/
-	framework.ConformanceIt("should support unsafe sysctls which are actually allowed [MinimumKubeletVersion:1.21]", func() {
+	framework.ConformanceIt("should forbidden unsafe sysctls [MinimumKubeletVersion:1.21]", func() {
```

Comment:
If changing the test purpose like this because of duplicated other test, it would be safer to drop this test from conformance test set.
Then we can check the test stability and make it promoted again, I feel.

/cc @oomichi